data/json/items/ammo.json

```diff
@@ -693,7 +693,7 @@
     "id": "120mm_HEAT",
     "price": "9 kUSD 999 USD 99 cent",
     "price_postapoc": "10 USD",
-    "name": { "str_sp": "120mm HEAT" },
+    "name": { "str": "120mm anti-tank rocket" },
```

Comment:
Not all explosive warheads are meant for the same purpose or target. This is equivalent to removing `FMJ` from a round's description.